content/articles/cancer-cell-classification-using-scikit-learn/index.md

```diff
@@ -0,0 +1,155 @@
+
+```bash
+pip install scikit-learn
+```
+> For this project, Jupyter notebooks are encouraged, although you are free to use any IDE you choose. Instead of developing an entire script from scratch, one can execute a few lines of code and see what happens one by one rather than writing the entire script and running it.
+
+To set it up, use the command line and enter the following code:
+
+```bash
+pip install jupyter
+```
+### The dataset
+We will be classifying cancer cells based on their features and identifying them if they are malignant or benign using scikit-learn library for python programming language. We will be using the breast cancer Wisconsin dataset for our machine learning problem.
+
+Malignant and benign breast cancer tumors are included in the data set, as well as their respective classification labels. The following function can be used to load it:
+
+```python
+load_breast_cancer([return_X_y])
+```
+The data set has data of 569 tumors and includes data on 30 attributes of a tumor that we will be using to train our model.
+
+### Classification implemented with the Scikit-learn framework
+#### 1. Adding the required modules and data to the import
+
+Scikit-learn and Breast cancer wisconsin (diagnostic) dataset will be imported into our program as a first step.
+
+```python
+import sklearn
+from sklearn.datasets import load_breast_cancer
+```
+#### 2. Creating a variable from the dataset
+We must take into account the labels, classification labels, feature meanings, and the data to be learned from this dataset.
+
+```python
+data = load_breast_cancer()
+```
+#### 3. Arranging and analyzing data
+Before we can learn how to train our model, we need to organize the data and and if you wish to what it contains then you can use the **print()** function in order to gain a better sense of what we have.
+
+```python
+namesForlabels = data['target_names']
+ourlabels = data['target']
+namesForfeature = data['feature_names']
+ourfeatures = data['data']
+```
+You can then use the **print()** method to examine the data.
+
+```python
+print(namesForlabels)
+```
+Output:
+
+```bash
+['malignant' 'benign']
+```
+Tumor data is classified as `malignant` or `non-malignant` as you can see.
+
+Labels 0 and 1 are binary values representing labels where 0 indicates malignant tumors and 1 indicates benign tumors when they are written.
+
+```python
+print(ourlabels)
+```
+The tumor's 30 distinct characteristics and properties may be seen in full detail when the features are printed. Training our model on factors such as a tumor's malignant or benign status will help us make an accurate prediction.
+
+```python
+print(ourfeatures)
+```
+There are 569 instances of tumor data in the dataset in question, each with a numerical value for one of the 30 variables in question. Data you have acquired indicates that the first tumor is malignant and has a mean radius of 1.79900000e+01 based on the information you know.

```

Comment:
Please include the outputs after running the `print` function before explaining the results. It makes your tutorial easy to follow.